packages/configs/default/index.json

```diff
@@ -1,7 +1,7 @@
 {
   "bundler": "@parcel/bundler-default",
   "transforms": {
-    "*.{js,mjs,jsm,jsx,es6}": [
+    "*.{js,mjs,jsm,jsx,es6,ts,tsx}": [
```

Comment:
I think typescript should be a separate transformer plugin, even if it uses babel internally. We'll be able to share ASTs anyway, so shouldn't be a perf hit.